src/sidebar.cls.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * The Sidebar class
+ *
+ * @since
+ * @package     LiteSpeed
+ * @subpackage  LiteSpeed/inc
+ * @author      LiteSpeed Technologies <info@litespeedtech.com>
+ */
+namespace LiteSpeed;
+
+defined( 'WPINC' ) || exit;
+
+class Sidebar extends Base {
+
+  /**
+   * Add a post meta box into the sidebar of post editor.
+   *
+   * @since    4.3
+   */
+  function add_meta_boxes() {
+    Debug2::debug( '[Sidebar] add_meta_boxes ');
+    add_meta_box(
+      'litespeed_metabox',                      // Unique ID
+      'Litespeed Cache',                        // Title
+      array( Sidebar::cls(), 'meta_box_html' ), // Callback function
```

Comment:
in class itself, can use `$this` while no need to reinitialize the class.

Reply:
it won't work if i use $this->meta_box_html()

Reply:
`array( $this, 'meta_box_html' )`

Reply:
That is just to register, not to run, surely can't use `$this->meta_box_html()` to hook

Reply:
I see, thanks

Reply:
got it thanks a lot